libs/community/langchain_community/vectorstores/kinetica.py

```diff
@@ -757,6 +763,7 @@ def from_texts(
         metadatas: Optional[List[dict]] = None,
         config: KineticaSettings = KineticaSettings(),
         collection_name: str = _LANGCHAIN_DEFAULT_COLLECTION_NAME,
+        schema_name: str = _LANGCHAIN_DEFAULT_SCHEMA_NAME,
```

Comment:
Could you update the ordering of parmeters in function signature here and in remaining places to avoid making a breaking change (see comment before about using `*` and placing at end of function signature)

Reply:
Nade this parameter keyword only as suggested